extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -307,9 +308,9 @@ private function addCurrentUserIfContributor( $allContributorsData, $currentUser
 		$key = array_search( $currentUserId, array_column( $allContributorsData, 'userId' ) );
 
 		if ( $key !== false ) {
-			$data = $allContributorsData[$key];
-			$data['isCurrent'] = true;
-			unset( $allContributorsData[$key] );
+			$data = $allContributorsData[ $key ];
+			$data[ 'isCurrent' ] = true;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "isCurrent" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)